Docs: Update events (#8625)
PostHog/posthog.com: contents/docs/data/events.mdx

---

> An event is the core unit of data in PostHog. It represents an interaction a user has with your app or website. Examples include button clicks, pageviews, query completions, and signups.

Reviewer: ```suggestion
Events represent an interaction a user has with your app or website. Examples include button clicks, pageviews, query completions, and signups.
```

> At its most basic, an event is: 

Reviewer: ```suggestion
Events consist of:
```

> 1. An `event` name like `$pageview` or `query completed`. Events starting with `$` are PostHog defaults.

Reviewer: ```suggestion
1. A `name`, like `$pageview` or `query completed`. Events starting with `$` events captured by PostHog, and not custom events.
```

> 2. A `distinct_id` which is a unique identifier for [person](/docs/data/person), commonly a `uuidv7` value like `018daf23-89b3-7cf8-a4f1-94064c96df90`

Reviewer: ```suggestion
2. A `distinct_id`, which is a unique identifier for [person](/docs/data/person). Commonly a `uuidv7` value like `018daf23-89b3-7cf8-a4f1-94064c96df90`
```

> 3. A `timestamp` in ISO 8601 format which is the time the event occurred like `2024-05-22T17:09:29.220Z`. This is most commonly `now()`, but can also be in the past. 

Reviewer: ```suggestion
3. A `timestamp` in ISO 8601 format, which is the time the event occurred like `2024-05-22T17:09:29.220Z`. This is most commonly `now()`, but can also be in the past. 
```

> 4. `properties` which are additional data like `color` or `$current_url`. Properties starting with `$` are PostHog defaults.

Reviewer: ```suggestion
4. `properties`, which are additional data like `color` or `$current_url`. Properties starting with `$` are PostHog defaults.
```

> You can capture custom events using any of [our SDKs](/docs/libraries) or [our API](/docs/api/capture). PostHog's client-side SDKs, like [JavaScript Web](/docs/libraries/js) and [React Native](/docs/libraries/react-native), can also [autocapture](/docs/data/autocapture) events for you.

Reviewer: ```suggestion
You can [capture custom events](/docs/product-analytics/capture-events) using any of [our SDKs](/docs/libraries) or [our API](/docs/api/capture). PostHog's client-side SDKs, like [JavaScript Web](/docs/libraries/js) and [React Native](/docs/libraries/react-native), can also [autocapture](/docs/data/autocapture) events for you.
```

You can view recent events in your [activity tab](https://us.posthog.com/events), which automatically updates every 30 seconds for a live view of activity in your app.

## How events power PostHog

PostHog is event-based, meaning much of the functionality is triggered through events. Much of this is handled (and made nicer) by our SDKs, but under the hood, it is largely events. For example:

- Changing [person properties](/docs/product-analytics/user-properties) using `$set` and `$unset` properties. 

> - Identifying an anonymous user with the `$identify` event.

Reviewer: ```suggestion
- [Identifying](/docs/product-analytics/identify) an anonymous user with the `$identify` event.
```

> - Connecting two users with the `$create_alias` event.

Reviewer: ```suggestion
- Connecting two users with the [`$create_alias`](/docs/product-analytics/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) event.
```

> - Creating and updating groups using `$groupidentify` events and `$group` properties. 

Reviewer: ```suggestion
- Creating and updating [groups](/docs/product-analytics/group-analytics) using `$groupidentify` events and `$group` properties. 
```

- Calculating bounce rate for web analytics relies on the `$pageleave` event.

- Calculating exposure and statistical significance for experimentation with the `$feature/experiment-feature-flag-key` property and `$feature_flag_called` event.

- Sending data to webhooks is triggered by ingesting specific events.

## Event properties

> Like persons and sessions, events have properties. These are used for [filtering](/docs/product-analytics/trends/filters), [breakdowns](/docs/product-analytics/trends/breakdowns), cohorts, targeting, and more.

Reviewer: ```suggestion
Like persons and sessions, events have properties. These are used for [filtering](/docs/product-analytics/trends/filters), [breakdowns](/docs/product-analytics/trends/breakdowns), [cohorts](/docs/data/cohorts), targeting, and more.
```
